frontend/resourceadm/components/ImportResourceModal/ServiceContent/ServiceContent.module.css

```diff
@@ -0,0 +1,4 @@
+.contentWrapper {
+  margin-top: 20px;
+  width: 80%;
```

Comment:
By using max-width: 480px and width: 100% it would scale better between screen sizes. 😊